sky/cli.py

```diff
@@ -1476,12 +1476,15 @@ def queue(clusters: Tuple[str], skip_finished: bool, all_users: bool):
     for cluster in clusters:
         try:
             job_table = core.queue(cluster, skip_finished, all_users)
-        except exceptions.NotSupportedError as e:
-            unsupported_clusters.append(cluster)
-            click.echo(str(e))
-            continue
-        except (RuntimeError, exceptions.ClusterNotUpError) as e:
-            click.echo(str(e))
+        except (RuntimeError, exceptions.NotSupportedError,
+                exceptions.ClusterNotUpError, exceptions.CloudUserIdentityError,
+                exceptions.ClusterOwnerIdentityMismatchError) as e:
+            if isinstance(e, exceptions.NotSupportedError):
+                unsupported_clusters.append(cluster)
+            click.echo(
+                f'{colorama.Fore.YELLOW}Failed to get the job queue for '
+                f'cluster {cluster!r}.{colorama.Style.RESET_ALL}\n'
+                f'  Details: [{common_utils.class_fullname(e.__class__)}] {e}')
```

Comment:
```suggestion
                f'  {e}')
```
Not feeling too strongly. Just that current output seems a bit verbose:
<img width="1585" alt="Screen Shot 2022-12-19 at 18 41 03" src="https://user-images.githubusercontent.com/592670/208407725-18b8775f-fb91-42a6-8dd0-d3ed4e516de0.png">

(Maybe it's due to the details are showing up in yellow as well?)

Reply:
I removed the color and `Details:` at the beginning, and it now looks like the following:
<img width="1002" alt="image" src="https://user-images.githubusercontent.com/6753189/208496932-5025a66f-16ad-4334-9ad4-8ba3cc6e1f60.png">

Reply:
LGTM